src/Chip/README.md

```diff
@@ -16,34 +16,82 @@ notes: |
 ## Basic Usage
 
 ```jsx live
-<div>
-  <Chip>New</Chip>
-  <Chip disabled>New</Chip>
-  <Chip variant="dark">New</Chip>
-</div>
+<Stack
+    gap={2}
+    direction="horizontal"
```

Comment:
We usually indent JS code by 2 spaces in this repository, can you please do that in this file as well for consistency?

Reply:
Good catch, thanks 👍